src/intrprtr.h

```diff
@@ -78,9 +72,9 @@
 **  return-void-statement was interpreted. If 'IntrEnd' returns 'STATUS_QUIT',
 **  then a quit-statement was interpreted.
 */
-void IntrBegin(Obj frame);
+void IntrBegin(IntrState * intr, Obj frame);
```

Comment:
Most (all?) of the remaining changes are about adding `IntrState * intr` as first parameter to `IntrFoobar` functions.